pkgs/development/compilers/sac2c/LICENSE.txt

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------
```

Comment:
Hi, the license should reflect https://gitlab.sac-home.org/sac-group/sac2c/-/blob/develop/LICENSE.md, as we're OSS now :smiley: